pkgs/by-name/we/wechat-uos/package.nix

```
@@ -0,0 +1,221 @@
+{ stdenvNoCC
+, stdenv
+, lib
+, fetchurl
+, requireFile
+, dpkg
+, nss
+, nspr
+, xorg
+, pango
+, zlib
+, atkmm
+, libdrm
+, libxkbcommon
+, xcbutilwm
+, xcbutilimage
+, xcbutilkeysyms
+, xcbutilrenderutil
+, mesa
+, alsa-lib
+, wayland
+, openssl_1_1
+, atk
+, at-spi2-atk
+, at-spi2-core
+, dbus
+, cups
+, gtk3
+, libxml2
+, cairo
+, freetype
+, fontconfig
+, vulkan-loader
+, gdk-pixbuf
+, libexif
+, ffmpeg
+, pipewire
+, libuuid
+, expat
+, bzip2
+, glib
+, systemd
+, libva
+, libGL
+, libnotify
+, buildFHSEnv
+, writeShellScript
+}:
```

New suggestion: this makes it easy for end user to supply the license via an `.override`

```suggestion
, /**
    License for wechat-uos, packed in a gz archive named "license.tar.gz".
    It should have the following files:

    license.tar.gz
    ├── etc
    │   ├── lsb-release
    │   └── os-release
    └── var
      ├── lib
      │   └── uos-license
      │       └── .license.json
      └── uos
          └── .license.key
  */
  uosLicense ? requireFile {
    name = "license.tar.gz";
    url = "https://www.uniontech.com";
    sha256 = "53760079c1a5b58f2fa3d5effe1ed35239590b288841d812229ef4e55b2dbd69";
  }
}:
```